{
  "path": "/documentation/cfnetwork/cfnetworkerrors/cfurlerrorsecureconnectionfailed",
  "type": "Enumeration Case",
  "name": "CFNetworkErrors.cfurlErrorSecureConnectionFailed",
  "desc": "The secure connection failed for an unknown reason.",
  "items": [],
  "declaration": "case cfurlErrorSecureConnectionFailed = -1200"
}